.github/workflows/nightly.yml

```diff
@@ -23,5 +22,5 @@ jobs:
       - run: yarn
       - run: ./scripts/pre-publish.sh --yes
         env:
-          VERSION: '0.0.0-${{ github.sha }}'
-          DIST_TAG: experimental
+          VERSION: '0.0.0-next.${{ github.sha }}'
```

Comment:
It would be nice if we could target a specific milestone though :thinking: 
For instance, the cloud only accepts > 4.7.0 projects, meaning that `@next` version couldn't be used on the cloud
I guess we could fetch & use the latest milestone but if we have multiple ones open at the same time we wouldn't know which one to use

Reply:
Milestone are not a great tool for this kind of things. We could just use the next patch version instead. Let's use next for now and we can improve later.